vspace/src/main/java/edu/asu/diging/vspace/web/staff/DeleteFileController.java

```diff
@@ -0,0 +1,28 @@
+package edu.asu.diging.vspace.web.staff;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.diging.vspace.core.services.impl.FileManager;
+
+@Controller
+public class DeleteFileController {
+    
+    @Autowired
+    private FileManager fileManager;
+    
+    @RequestMapping(value = "/staff/files/delete/{fileId}", method = RequestMethod.POST)
```

Comment:
all  the  urls should  be `.../files/{fileId}/<action>` e.g. `.../files/{fileId}/delete`